source/databases-collections.txt

```diff
@@ -0,0 +1,286 @@
+cursor containing all collections in the database and their associated metadata.
+
+The following example calls the ``listCollections()`` method and iterates over
+the returned cursor to print the collections from the :ref:`php-db-coll-access-collection`
+and :ref:`php-db-coll-create-collection` examples:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/databases-collections/databases-collections.php
+      :language: php
+      :start-after: start-find-collections
+      :end-before: end-find-collections
+      :dedent:
+
+   .. output::
+      :language: console
+      :visible: false
+
+      MongoDB\Model\CollectionInfo Object
+      (
+         [name] => example_collection
+         [type] => collection
+         ...
+      )
+      MongoDB\Model\CollectionInfo Object
+      (
+         [name] => test_collection
+         [type] => collection
+         ...
+      )
+
+For more information about iterating over a cursor, see the :ref:`php-cursors`
+guide. 
+
+Delete a Collection
+-------------------
+
+You can delete a collection from the database by using the ``MongoDB\Database::dropCollection()``
+method.
+
+The following example deletes the ``test_collection`` collection:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-drop-collection
+    :end-before: end-drop-collection
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database permanently deletes all
+   documents and all indexes within that collection.
+
+   Drop a collection only if you no longer need the data in it.
+
+.. _php-config-read-write:
+
+Configure Read and Write Operations
+-----------------------------------
+
+You can control how the library routes read operations by setting a **read preference**.
+You can also control options for how the library waits for acknowledgment of
+read and write operations on a replica set by setting a **read concern** and a
+**write concern**.
+
+By default, databases inherit read and write settings from the ``MongoDB\Client``
+instance, and collections inherit them from the database. You can change
+these settings on your database by passing an options array to the
+``MongoDB\Client::selectDatabase()``. You can change these settings on your collection
+by passing an options array to the ``MongoDB\Client::selectCollection()`` method.
+
+To change the read preference, read concern, and write concern of your database
+or collection, set the following options in the array parameter:
+
+- ``readPreference``: Sets the read preference. For a list of available read
+  preferences, see `MongoDB\\Driver\\ReadPreference <{+php-manual+}/class.mongodb-driver-readpreference>`__
+  in the extension API documentation.
+- ``readConcern``: Sets the read concern. For a list of available read
+  concerns, see `MongoDB\\Driver\\ReadConcern <{+php-manual+}/class.mongodb-driver-readconcern>`__
```

Comment:
Existing docs use the `:php:` directive. For example:

```
:php:`MongoDB\Driver\ReadConcern <class.mongodb-driver-readconcern>`
```

Are there plans to eliminate that? If not, should you use it for PHP.net links?

Reply:
Good to know, I wasn't aware of that directive - updated